source/includes/fundamentals/code-snippets/SearchText.java

```diff
@@ -0,0 +1,89 @@
+package docs;
+
+import com.mongodb.client.MongoClient;
+import com.mongodb.client.MongoClients;
+import com.mongodb.client.MongoCollection;
+import com.mongodb.client.MongoDatabase;
+
+import org.bson.Document;
+import org.bson.conversions.Bson;
+
+import java.util.Arrays;
+
+import com.mongodb.client.model.Filters;
+import com.mongodb.client.model.Indexes;
+import com.mongodb.client.model.Projections;
+import com.mongodb.client.model.Sorts;
+
+public class SearchText {
+    
+    private final MongoCollection<Document> collection;
+    private final MongoClient mongoClient;
+    private final MongoDatabase database;
+
+    private SearchText() {
+        final String uri = System.getenv("DRIVER_REF_URI");
+
+        mongoClient = MongoClients.create(uri);
+        database = mongoClient.getDatabase("crudOps");
+        collection = database.getCollection("searchText");
+    }
+
+    public static void main(String [] args){
+        SearchText searchText = new SearchText();
+
+        searchText.setupCoursesCollection();
+        // System.out.println("Matches for 'Fast'");
+        // searchText.termExample();
+
+        // System.out.println("Matches for 'Fate 7'");
+        // searchText.multipleTermExample();
+
+        System.out.println("Matches for 'Furious' without 'Fast'");
+        searchText.negateExample();
+
+        // System.out.println("Matches for 'Fate of the Furious'");
+        // searchText.phraseExample();
+    }
+
+    private void termExample(){
+        // begin termExample
+        Bson filter = Filters.text("Fast");
```

Comment:
Nitpick: since you're specifying "fast" as the term in a case-insensitive search, it might make more sense to keep it lowercase rather than capitalize it:
Related comment: https://github.com/mongodb/docs-java/pull/85/files/a682aaaa0f606d5516304e16e46eb0b9b0e28b00#r646777048